components/apimgt/org.wso2.carbon.apimgt.keymgt/src/test/java/org/wso2/carbon/apimgt/keymgt/issuers/PermissionBasedScopesIssuerWrapper.java

```diff
@@ -0,0 +1,59 @@
+package org.wso2.carbon.apimgt.keymgt.issuers;
```

Comment:
Missing license header

Reply:
Fixed in 121cc9e994e389d43cc4b11ccabf7bddeadebc91.